README.md

```diff
@@ -2,6 +2,25 @@
 
 A [Jabba](https://github.com/shyiko/jabba)-compatible index of JDK versions
 
+## Availability of JDK versions
+This index aims to provide only up-to-date versions of the JDK. Any non-LTS versions will be removed soon after reaching
+EOL (usually with the release of the next version of the JDK). EOL releases **do not** receive security or bug fix
+updates and there is **no reason** to stay on them, people are very much encouraged to upgrade. Soon after publishing
+the latest builds for the current JDKs, they will replace the old, out-of-date build (this usually happens every 3
+months). There is also no possibility to select a specific build of the JDK (unless going through the commit history and
+depending on a specific version in time of this index). This is a common practice with the new release schedule of JDKs
+and users should be aware by now.
```

Comment:
This has been bothering me, as it means irreproducible builds.  But I generally want to stay on the latest patch, and there's no obvious automated update mechanism to point at the latest git hash of this repo.

I think this text is right, but The Way We Do Things isn't.

Reply:
Before you merge, I will write some more info for your consideration.

Reply:
Even today, as we usually use `ThisBuild / githubWorkflowJavaVersions` with strings like `adoptium@8`, when Jabba runs with that string as input, it still selects the latest available JDK from the index, that matches that string. For example, if you have JDK 1.8.292 and JDK 1.8.302 in the index, and you try to re-run a build from 6 months ago that used 1.8.292, it would still switch over to 1.8.302 in the re-run. A lot of things suck about this, I agree.

Reply:
We can keep all versions of Java forever, almost like the original jabba index tried, but then we also need to demand more discipline when selecting the specific, reproducible version. It would have to be changed to `adoptium@1.8.302` for example, to select that and only that version.

This index has still very few users (Cats Effect, disciplines-specs2 which I maintain and http4s), so the time for changes is now.

Reply:
On the other hand, people have to accept that Java is now a rolling release. A bug reproducible on an EOL version (like 9, 10, 12, 13, 14, 15, 16) or a specific release (like 1.8.292, 11.0.11) and not reproducible on a newer, current version (8, 11, 17) or release (1.8.302, 11.0.12), maintainers don't really have to care or provide support for those types of issues.

Unfortunately, I also do understand the importance of build reproducibility.

Reply:
Yes, you're right.  I'm not throwing a bigger fit about it because the use of `adoptium@8` before this index already put us in this position, and the culture of the JVM tolerates this.  It was only a major problem in practice once for me: blaze had SSL bugs on a subset of Java 11 versions.  It's tolerable because the JVM does such a great job on compatibility. 

If we did pin to a specific version of either the JDK (if this supported patches), or to a specific commit in this repo, we'd also want a "JDK steward" to provide automation to keep it fresh.  It would be a chore to simulate the rolling release we're getting with this approach for the very rare time we need to stop the roll.  Projects that _really_ need that should look at Nix+Niv instead.  This should be good for most people.

Reply:
Jabba does support patches, we can go that route. It just requires some discipline in the selection strings.

Reply:
I think we are good to roll forward as is.  This is a good 99% solution, and there are other good solutions for the 1%.

Reply:
Alright, pressing forward with the change.